src/devrules/cli_commands/branch.py

```
@@ -239,57 +239,77 @@ def delete_branch(
+                    header="🗑 Delete Branches",
+                    subheader="📋 Select branches to be deleted:",
+                    messages=[f"{idx}. {b}" for idx, b in enumerate(owned_branches, 1)],
+                )
+                typer.echo()
+                choices = typer.prompt("Enter number, multiple separated by a space", type=str)
+                choices = choices.split(" ")
+                try:
+                    choices = [int(choice) for choice in choices]
+                except ValueError:
+                    typer.secho(msg.INVALID_CHOICE, fg=typer.colors.RED)
+                    raise typer.Exit(code=1)
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)